manifests/init.pp

```diff
@@ -31,6 +32,14 @@
     source  => 'puppet:///modules/pe_metric_curl_cron_jobs/tk_metrics'
   }
 
+  file { '/opt/puppetlabs/bin/puppet-metrics-collector':
```

Comment:
`/opt/puppetlabs/bin` doesn't appear to be in the $PATH by default.  Is it beneficial to put this there?  

```
[root@master201646-centos ~]# ls -l /opt/puppetlabs/bin/
total 144
lrwxrwxrwx 1 root root     52 Jul 21 23:55 console-services -> ../server/apps/console-services/bin/console-services
lrwxrwxrwx 1 root root     33 Jul 21 23:53 facter -> /opt/puppetlabs/puppet/bin/facter
lrwxrwxrwx 1 root root     32 Jul 21 23:53 hiera -> /opt/puppetlabs/puppet/bin/hiera
lrwxrwxrwx 1 root root     30 Jul 21 23:53 mco -> /opt/puppetlabs/puppet/bin/mco
lrwxrwxrwx 1 root root     64 Jul 21 23:55 orchestration-services -> ../server/apps/orchestration-services/bin/orchestration-services
lrwxrwxrwx 1 root root     33 Jul 21 23:53 pe-man -> /opt/puppetlabs/server/bin/pe-man
lrwxrwxrwx 1 root root     33 Jul 21 23:53 puppet -> /opt/puppetlabs/puppet/bin/puppet
lrwxrwxrwx 1 root root     46 Jul 21 23:53 puppet-access -> /opt/puppetlabs/client-tools/bin/puppet-access
lrwxrwxrwx 1 root root     43 Jul 21 23:53 puppet-app -> /opt/puppetlabs/client-tools/bin/puppet-app
lrwxrwxrwx 1 root root     44 Jul 21 23:53 puppet-code -> /opt/puppetlabs/client-tools/bin/puppet-code
lrwxrwxrwx 1 root root     36 Jul 21 23:55 puppetdb -> ../server/apps/puppetdb/bin/puppetdb
lrwxrwxrwx 1 root root     42 Jul 21 23:53 puppet-db -> /opt/puppetlabs/client-tools/bin/puppet-db
lrwxrwxrwx 1 root root     60 Jul 21 23:55 puppet-enterprise -> /opt/puppetlabs/server/apps/enterprise/bin/puppet-enterprise
-rwxr-xr-x 1 root root 136894 Jul 21 23:53 puppet-enterprise-uninstaller
lrwxrwxrwx 1 root root     43 Jul 21 23:53 puppet-job -> /opt/puppetlabs/client-tools/bin/puppet-job
lrwxrwxrwx 1 root root     45 Jul 21 23:53 puppet-query -> /opt/puppetlabs/client-tools/bin/puppet-query
lrwxrwxrwx 1 root root     44 Jul 21 23:53 puppetserver -> ../server/apps/puppetserver/bin/puppetserver
lrwxrwxrwx 1 root root     31 Jul 21 23:54 r10k -> /opt/puppetlabs/puppet/bin/r10k
[root@master201646-centos ~]# echo $PATH
/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bin:/root/bin
[root@master201646-centos ~]# ls -l /usr/local/bin/
total 0
lrwxrwxrwx 1 root root 26 Jul 21 23:54 facter -> /opt/puppetlabs/bin/facter
lrwxrwxrwx 1 root root 25 Jul 21 23:54 hiera -> /opt/puppetlabs/bin/hiera
lrwxrwxrwx 1 root root 23 Jul 21 23:54 mco -> /opt/puppetlabs/bin/mco
lrwxrwxrwx 1 root root 26 Jul 21 23:54 pe-man -> /opt/puppetlabs/bin/pe-man
lrwxrwxrwx 1 root root 26 Jul 21 23:54 puppet -> /opt/puppetlabs/bin/puppet
lrwxrwxrwx 1 root root 24 Jul 21 23:54 r10k -> /opt/puppetlabs/bin/r10k
```